gdk/PixbufLoader.custom

```diff
@@ -47,14 +49,18 @@
 			return this.Write (bytes, (ulong) count);
 		}
 
-		private void LoadFromStream (System.IO.Stream input)
-		{
-			byte [] buffer = new byte [8192];
-			int n;
+        const int bufSize = 8192;
+        static ByteArrayPool pool = new ByteArrayPool ();
```

Comment:
Would it make more sense to add an overload so the caller can supply a buffer rather than making the toolkit cache buffers? Then API consumers can opt to cache and supply whatever buffers they want.